client/src/components/Info.jsx

```diff
@@ -1,35 +1,32 @@
-import React from 'react';
+import React, { Component } from 'react';
 
-import Star from './Star.jsx';
+import Rating from './Rating.jsx';
+import AdditionalInfo from './AdditionalInfo.jsx';
 
-class Info extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
+const Info = function(props) {
+  var editorsChoiceLogo = '';
+  var editorsChoice = '';
 
-    }
+  if (props.app.editorChoice === true) {
+    editorsChoiceLogo = 'https://lh3.googleusercontent.com/HotsP0KmK4tn0Q8p9szRXtwjD7fZRKu4mFcfJUFoddrGiZefxY7gz4dEGMuH6HsfCymJP6a8MvAwYWrU=s14-rw';
```

Comment:
This is interesting -- I had no idea that you could do this.  I would've thought you would have to use an img tag for this.  Is there a particular reason you chose to render this way instead of an img tag?

Reply:
I only did it this way because the data that I got back from the server regarding the editor's choice was a boolean. This I felt was the easiest way for me to "toggle" the image depending on what the response was.